lib/UserBackend.php

```diff
@@ -21,6 +21,8 @@
 
 namespace OCA\User_SAML;
 
+use OC\User\Backend;
+use OCA\User_LDAP\FilesystemHelper;
```

Comment:
Don't use this – it belongs to another app and might not be available, thus breaking user_saml.